cmd/meroxa/root/apps/describe.go

```diff
@@ -52,14 +48,6 @@ type Describe struct {
 	args struct {
 		NameOrUUID string
 	}
-
-	flags struct {
-		Extended bool `long:"extended" usage:"whether to show additional details about the Turbine Data Application"`
```

Comment:
I don't think we should remove support from a flag just like this. @janelletavares have you checked CLI metrics to see if we could confidently do this without putting this under a deprecation warning period?

Without any data that tells me one way or another, my suggestion would be to leave this functionality as is and only add new functionality first.

Reply:
I just checked metrics on that command and it seems we've gotten any usage with that flag so it's safe to remove it for now without going through the deprecation process.